client/scss/layouts/_account.scss

```diff
@@ -1,29 +1,23 @@
 .avatar-panel {
-  margin-top: 20px;
-
   &__image {
-    float: left;
-    width: 16%;
-    margin-bottom: 20px;
+    .avatar-clear:checked ~ .avatar.avatar-span {
+      display: none;
+    }
+
+    .avatar-clear:not(:checked) ~ .avatar.gravatar {
+      display: none;
+    }
   }
 
   &__form {
-    float: left;
-
     label {
       display: block;
       padding: 0;
       width: unset;
-      float: unset;
       margin-bottom: 10px;
     }
   }
 
-  &__revert-to-gravatar {
-    margin-top: 20px;
-    margin-bottom: 20px;
-  }
-
```

Comment:
If we are no longer using floats. We can remove the after content below lines 21-25.